ToDo/ToDoTableViewController.swift

```diff
@@ -34,14 +34,25 @@ class ToDoTableViewController: UITableViewController {
     }
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "ToDoCellIdentifier", for: indexPath)
-        
+        let cell = tableView.dequeueReusableCell(withIdentifier: "ToDoCellIdentifier", for: indexPath) as! ToDoCell
+                
         let toDo = toDos[indexPath.row]
-        cell.textLabel?.text = toDo.title
+        cell.titleLabel?.text = toDo.title
+        cell.isCompleteButton.isSelected = toDo.isComplete
+        cell.delegate = self
         
         return cell
     }
     
+    func checkmarkTapped(sender: ToDoCell) {
+        if let indexPath = tableView.indexPath(for: sender){
+            var todo = toDos[indexPath.row]
+            todo.isComplete.toggle()
+            toDos[indexPath.row] = todo
+            tableView.reloadRows(at: [indexPath], with: .automatic)
+        }
+    }
+    
     @IBAction func unwindToToDoList(segue: UIStoryboardSegue){
         guard segue.identifier == "saveUnwind" else { return }
         let sourceViewController = segue.source as! ToDoDetailTableViewController
```

Comment:
guard-ը հեն ա գրել ես էլի, միանգամից էս էլ մտցրու մեջը, էլ force unwrap մի արա

Reply:
guardi mej mcneluc error a talis

Reply:
as!-ը սարքել ես չէ՞ as?

Reply:
guard segue.identifier == "saveUnwind",
              let sourceViewController = segue.source as ToDoDetailTableViewController
        else { return }
senc em grel error a talis " 'UIViewController' is not convertible to 'ToDoDetailTableViewController' "

Reply:
as? գրի

Reply:
պոլիմորֆիզմի պատճառով դու, չես կարա նախօրոք իմանաս, թե ինչ տիպի ա, դու մենակ գիտես, որ ինչ որա ա UIViewController-ի ժառանգ ա, ասում ես հլը փորձի ToDoDetailTableViewController ա՞, էդ as? ի պահն ա, պլյուս որ ուզում ես հանկարծ ուրիշ բան չլինի, դնում ես գարդի մեջ

Reply:
Haa lav haskaca